hnn_core/network.py

```diff
@@ -1419,9 +1420,12 @@ def plot_cells(self, ax=None, show=True):
         """
         return plot_cells(net=self, ax=ax, show=show)
 
-    @copy_doc(write_network)
-    def write(self, fname, overwrite=True, write_output=True):
-        write_network(self, fname, overwrite, write_output)
+    def to_dict(self, write_output=False):
```

Comment:
why is it necessary to have two names for the same function?

Reply:
I think this was just an organizational thing to keep the IO logic in hnn_io because the network module was getting big. That said, I suppose to_dict isn't really IO, it's just mostly used for IO operations. The logic can just be moved to this module if you prefer.